skribenten-web/frontend/vite.config.ts

```diff
@@ -1,3 +1,4 @@
+/// <reference types="vitest" />
```

Comment:
Fjerne kommentar

Reply:
Det er ikke en kommentar 😛  Det er kontekst config fila trenger for å få type definisjoner til vitest config